### DISCUSSION TITLE
Let me know what you're using this for

### DISCUSSION THREAD

**Discussion by gavanderhoorn** [gavanderhoorn/dominh#11] [General]
If you've found `dominh` useful, post a quick note.

If something doesn't work, open an [issue](https://github.com/gavanderhoorn/dominh/issues) instead :)

**Comment by AnAverageSloth** [comment]
Great stuff man. I'm using this for a robot vision system on a 39ib+. Not being able to write to pos regs requires me to do some internal logic on the controller, but still awesome. I created an account just to say this. Thanks and great stuff.

**Reply by gavanderhoorn** [reply]
Thanks. Good to know the library helps.

> Not being able to write to pos regs requires me to do some internal logic on the controller, but still awesome

Yeah, I might add this sometime next month. I've been getting more-and-more annoyed by this as well, but so far I haven't been able to find a non-Karel way to do this -- and as I noted in the readme, I'd like to avoid using Karel programs (or any non-standard options really), unless absolutely necessary.

But I might just accept using Karel for now. It's not that difficult to write something for this.

---

Edit: tracking this in #3 already btw.

**Comment by pdragy** [comment]
I've used this to try and bootstrap fanuc robot variables, and also for general debugging from linux. It works great!  I've also used the CLI on windows along with [fexcel](https://github.com/onerobotics/fexcel) for managing controller setup. I tried extending the CLI a while back for convenience to include 'set' and 'info' commands, would this be something you would want to upstream? [https://github.com/WILDER-SYSTEMS-LLC/dominh/commit/b5618f1](https://github.com/WILDER-SYSTEMS-LLC/dominh/commit/b5618f1)

**Reply by gavanderhoorn** [reply]
thanks for your comment :+1: 

> I tried extending the CLI a while back for convenience to include 'set' and 'info' commands, would this be something you would want to upstream? [WILDER-SYSTEMS-LLC/dominh/commit/b5618f1](https://github.com/WILDER-SYSTEMS-LLC/dominh/commit/b5618f1)

I would definitely be interested in your modifications.

Can't look at them though as it appears it's a private fork?

**Reply by pdragy** [reply]
Sorry about that, I opened a PR here, maybe we can move the discussion over.
https://github.com/gavanderhoorn/dominh/pull/25

> thanks for your comment 👍
> 
> > I tried extending the CLI a while back for convenience to include 'set' and 'info' commands, would this be something you would want to upstream? [WILDER-SYSTEMS-LLC/dominh/commit/b5618f1](https://github.com/WILDER-SYSTEMS-LLC/dominh/commit/b5618f1)
> 
> I would definitely be interested in your modifications.
> 
> Can't look at them though as it appears it's a private fork?
